### PR TITLE
AWS Glue 1.0

### PR DESCRIPTION
**Pull request by Galaxy0229**
This is a simple intro of AWS Glue

### CODE REVIEW

**Comment by jimschenchen** [comment]
Could you elaborate on each steps? It is little bit hard to follow.

**Comment by Galaxy0229** [comment]
> Could you elaborate on each steps? It is little bit hard to follow.

Okay I will give steps more in detail

**Comment by Galaxy0229** [comment]
> Could you elaborate on each steps? It is little bit hard to follow.

Oops and I actually did not include the steps that I wrote, mb lol.

**Comment by Galaxy0229** [comment]
> Could you elaborate on each steps? It is little bit hard to follow.

Hey Chenyue, I updated the steps' details. Does them look good for u? Anything to improve?

**Review by jimschenchen — APPROVED** [review]
LGTM
